### PR TITLE
Add carolinar7 to kubernetes.

### PR DESCRIPTION
**Pull request by carolinar7**
fixes: https://github.com/kubernetes/org/issues/3550

/area github-membership

### CODE REVIEW

File: config/kubernetes/org.yaml [hunk 1]
```diff
@@ -191,6 +191,7 @@ members:
 - carlory
 - carlosedp
 - carolynvs
+- carolinar7
```
Comment (mrbobbytables) [inline config/kubernetes/org.yaml:194]: The org membership list is alpha sorted^^;;; Could you move this up?

Comment (carolinar7) [reply config/kubernetes/org.yaml]: Updated!

**Review by mrbobbytables — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**